### PR TITLE
[Django 3.0] Adding compatible versions of other libraries according …

### PR DESCRIPTION
**Pull request by agl29**
…to Django==3.0.12

## What changes were proposed in this pull request?

Upgrading other packages version as per requirement of Django==3.0.12
## How was this patch tested?
Screenshot 2021-02-17 at 10 00 24 PM

### CODE REVIEW

**Comment by agl29** [comment]
But now, django_babel==0.6.2 is the latest version And it supports only till Django2. SS
Screenshot 2021-02-17 at 10 01 27 PM
You can see this in CI also.

**Review by romainr — APPROVED** [review]
:+1: We can fix the issues in follow-up when making the Web Sockets GA

**Comment by romainr** [comment]
ERROR: Cannot install -r /root/repo/desktop/core/requirements.txt (line 11), -r /root/repo/desktop/core/requirements.txt (line 13), -r /root/repo/desktop/core/requirements.txt (line 14), -r /root/repo/desktop/core/requirements.txt (line 7) and Django==3.0.12 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested Django==3.0.12
    channels 3.0.3 depends on Django>=2.2
    django-auth-ldap 1.3.0 depends on django>=1.8
    django-axes 5.13.0 depends on django>=2.2
    django-babel 0.6.2 depends on django<3.0 and >=1.8

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

**Comment by romainr** [comment]
Upgrade 

django-babel 0.6.2 --> to remove 

FYI: could also use one the fork does not work
https://github.com/python-babel/django-babel/pull/49

You can point to a git directly, e.g.

    ./build/env/bin/pip install  git+https://github.com/gethue/PyHive

File: desktop/core/requirements.txt [hunk 1]
```diff
@@ -64,3 +63,4 @@ sqlparse==0.4.1
 tablib==0.13.0
 thrift==0.13.0
 thrift-sasl==0.4.2
+git+git://github.com/agl29/django-babel.git
```
Comment (agl29) [inline desktop/core/requirements.txt:66]: can we do this @romainr?

Comment (romainr) [reply desktop/core/requirements.txt:66]: :+1: 
And we could move it to http://github.com/gethue at the end of Django 3 if the lib is still needed

**Review by romainr — APPROVED** [review]

**Comment by agl29** [comment]
```
======================================================================
ERROR: oozie.tests.TestEditor.test_get_coordinator_parameters
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/repo/build/env/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/root/repo/apps/oozie/src/oozie/tests.py", line 1907, in test_get_coordinator_parameters
    coord = create_coordinator(self.wf, self.c, self.user)
  File "/root/repo/apps/oozie/src/oozie/tests.py", line 4026, in create_coordinator
    response = client.get(reverse('oozie:create_coordinator'))
  File "/root/repo/build/env/lib/python3.6/site-packages/django/test/client.py", line 518, in get
    response = super().get(path, data=data, secure=secure, **extra)
  File "/root/repo/build/env/lib/python3.6/site-packages/django/test/client.py", line 346, in get
    **extra,
  File "/root/repo/build/env/lib/python3.6/site-packages/django/test/client.py", line 421, in generic
    return self.request(**r)
  File "/root/repo/build/env/lib/python3.6/site-packages/django/test/client.py", line 496, in request
    raise exc_value
  File "/root/repo/build/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/root/repo/build/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/repo/build/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/root/repo/apps/oozie/src/oozie/decorators.py", line 145, in decorate
    return view_func(request, *args, **kwargs)
TypeError: create_coordinator() got an unexpected keyword argument 'bundle'
```
For this type of error, I was trying to understand that what changes between python2 and python3 but not found as of now.

**Comment by romainr** [comment]
Quick hint, seems to come from:

`    response = client.get(reverse('oozie:create_coordinator'))`

File: apps/oozie/src/oozie/views/editor.py [hunk 2]
```diff
@@ -398,7 +398,7 @@ def schedule_workflow(request, workflow):
 
 
 @check_job_access_permission()
-def create_coordinator(request, workflow=None):
+def create_coordinator(request, workflow=None, *args, **kwargs):
```
Comment (romainr) [inline apps/oozie/src/oozie/views/editor.py:401]: Feels kind of invalid but this code should not be used anymore

**Review by romainr — APPROVED** [review]

**Pull request merged**